e2e/tests/init_test.go

```diff
@@ -193,6 +193,10 @@ func (s *suite) TheUserStakesAWithUpoktFromTheAccount(actorType string, amount i
 	require.NoError(s, err, "error creating config file in %q", path.Join(os.TempDir(), configPathPattern))
 
 	configContent := fmt.Sprintf(`stake_amount: %d upokt`, amount)
+	if actorType == "application" {
```

Comment:
Thoughts on:

1. Adding `service` to the function name?
2. If the `actorType` == `gateway`, create an assertion that says "must be empty"
3. We can reuse this for suppliers & applications
 -> Will need to update some of the other tests

Reply:
1. Adding `service` to the function name will make it sound unnatural for the `gateway` case. I prefer either:
  * Have another function for `gateway`s that wraps this one and omit the `service` part
  * Just add the service variable before the actor type which sounds:
    * `the user stakes a "svc" "application" with "1000070" uPOKT from the account "app2"
    * `the user stakes a "gateway" with "1000070" uPOKT from the account "gateway2"
    
2. :+1: 
3. :+1:

Reply:
Let's create two separate functions and use this structure: `the user stakes "application" with "1000070" uPOKT for "service" from the account "app2"`